src/Manager/ManagerBase.php

```diff
@@ -40,4 +54,28 @@ protected function getResponseJson(RequestInterface $request)
         return json_decode($body, true);
     }
 
+    /**
+     * Get query string of using the options.
+     *
+     * @param $options The options
+     * @return string  The query string
+     */
+    protected function getQueryString($options) {
+      $queries = [];
+      foreach ($this->queryParameters as $queryName => $queryDefaultValue) {
+        // Use user value if possible.
+        if (isset($options[$queryName])) {
+          $queries[] = $queryName . '=' . $options[$queryName];
```

Comment:
Nice catch. I do need to worry about it. Done @ ea2057291ad686cbe7b3c4cc0fab6e94cd0d7e47.